### PR TITLE
Review 'Reuse directories of content'

### PR DESCRIPTION
**Pull request by jdbaldry**
- Avoid some passive voice
- Correct name of Hugo feature
- Make some clarifying edits

The rest LGTM

Closes https://github.com/grafana/writers-toolkit/issues/753

### CODE REVIEW

File: docs/sources/write/reuse-content/reuse-directories/index.md [hunk 1]
```diff
@@ -9,32 +9,31 @@ keywords:
...
 # Reuse directories of content with Hugo mounts
 
 {{< admonition type="note" >}}
-The following task can only be performed by a Grafana Labs employee.
+Only a Grafana Labs employee can perform the following task.
 {{< /admonition >}}
 
-Before Hugo performs a build, _Hugo mounts_ mount a source directory at a destination directory.
-Use Hugo mounts to reuse a whole directory of content in another part of the published technical documentation.
+Before Hugo performs a build, [_Hugo module mounts_](https://gohugo.io/hugo-modules/configuration/#module-configuration-mounts) mount a source directory at a destination directory.
```
Comment (clayton-cornell) [inline docs/sources/write/reuse-content/reuse-directories/index.md:22]: This sentence still reads oddly... should it be `mounts a source directory`?  (even that way it's still sounding odd to me)

Comment (jdbaldry) [reply docs/sources/write/reuse-content/reuse-directories/index.md:22]: Yeah I actually spent a bit of time trying to reword it and then left it.

What about:

```suggestion
Before Hugo performs a build, [_Hugo module mounts_](https://gohugo.io/hugo-modules/configuration/#module-configuration-mounts) configure a virtual filesystem.
Each mount mounts a source directory at a destination directory within that filesystem.
```

Comment (clayton-cornell) [reply docs/sources/write/reuse-content/reuse-directories/index.md:22]: Ah, that makes a LOT more sense.

**Review by clayton-cornell — APPROVED** [review]
Approved with the wording change as discussed

**Pull request merged**